### PR TITLE
[file-io-light] define a theory useful for guard-verified file output…

### PR DESCRIPTION
**Pull request by bendyarm**

### CODE REVIEW

File: books/kestrel/file-io-light/output-theory-test.lisp [hunk 1]
```diff
@@ -0,0 +1,100 @@
+; Simple regresstion tests for output-theory.lisp
```
Comment (acoglio) [inline books/kestrel/file-io-light/output-theory-test.lisp:1]: regression

File: books/kestrel/file-io-light/top.lisp [hunk 2]
```diff
@@ -77,4 +77,8 @@
 (include-book "file-length-dollar")
 (include-book "file-is-newer-thanp")
 
+;; Books grouped together for a specific purpose:
+(include-book "output-theory")
+(include-book "output-theory-test")
```
Comment (acoglio) [inline books/kestrel/file-io-light/top.lisp:82]: I wouldn't include test files into top files. The main purpose of top files is to ensure the absence of name conflicts, but that doesn't matter for tests.

**Review by acoglio — APPROVED** [review]
I'm not familiar with these rules, but the PR looks reasonable as far as I can tell.

File: books/kestrel/file-io-light/top.lisp (hunk 2 above)
Comment (bendyarm) [reply books/kestrel/file-io-light/top.lisp]: Done, thanks!

File: books/kestrel/file-io-light/output-theory-test.lisp (hunk 1 above)
Comment (bendyarm) [reply books/kestrel/file-io-light/output-theory-test.lisp]: fixed.

**Pull request closed without merging**